xml/art_sles_amd-sev.xml

```diff
@@ -0,0 +1,431 @@
+       <para>
+        Sending the guest to another platform is disallowed when set
+       </para>
+      </entry>
+     </row>
+     <row>
+      <entry>
+       <para>4</para>
+      </entry>
+      <entry>
+       <para>
+        The guest must not be transmitted to another platform that is not in the
+        domain when set
+       </para>
+      </entry>
+     </row>
+     <row>
+      <entry>
+       <para>5</para>
+      </entry>
+      <entry>
+       <para>
+        The guest must not be transmitted to another platform that is not
+        SEV-capable when set
+       </para>
+      </entry>
+     </row>
+     <row>
+      <entry>
+       <para>6-15</para>
+      </entry>
+      <entry>
+       <para>
+        Reserved
+       </para>
+      </entry>
+     </row>
+     <row>
+      <entry>
+       <para>16-32 </para>
+      </entry>
+      <entry>
+       <para>
+        The guest must not be transmitted to another platform with a lower
+        firmware version
+       </para>
+      </entry>
+     </row>
+    </tbody>
+    </tgroup>
+  </table>
+  <para>
+   The optional <literal>&lt;dhCert&gt;</literal> element provides the guest
+   owner's base64-encoded Diffie-Hellman (DH) key. The key is used to negotiate
+   a master secret key between the SEV firmware and guest owner. This master
+   secret key is then used to establish a trusted channel between SEV firmware
+   and guest owner.
+  </para>
+  <para>
+   The optional <literal>&lt;session&gt;</literal> element provides the guest
+   owner's base64-encoded session blob, as defined in the SEV API spec. See the
+   LAUNCH_START section of the SEV specification for the session-blob format.
+  </para>
+  <para>
+   In addition to the <literal>&lt;launchSecurity&gt;</literal> settings,
+   SEV-encrypted VMs must have the <literal>iommu='on'</literal> attribute set
+   in each <literal>virtio</literal> device, as shown in the example
+   configuration. This attribute is required in order to enable DMA APIs for
+   the device within QEMU.
+  </para>
+  <para>
+   SEV-encrypted VMs must also have all of their memory regions locked to allow
+   DMA and prevent swapping. This includes not only the VM's RAM, but also
+   ROM(s), pflash, and video memory. Using the example configuration, the memory
+   regions needing to be locked total to 4352592 kB, which exceeds the default
+   <literal>memlock</literal> limit. The <literal>memlock</literal> limit for a
+   virtual machine process can be increased using the <literal>&lt;hard_limit&gt;</literal>
+   subelement of <literal>&lt;memtune&gt;</literal>. The value 4352592 kB was
+   determined using the following formula:
+  </para>
+  <screen>hard_limit = VM RAM + VM video memory + ROMs/ACPI tables</screen>
+  <para>
+   where:
+  </para>
+<screen>    4096 kB (UEFI ROM)
+ +  4096 kB (UEFI variable store)
+ +   128 kB (PC ROM)
+ +   128 kB (ISA BIOS) 
+ +  2384 kB (ACPI tables)
+ = 10832 kB (total of all ROMs/ACPI tables)</screen>
```

Comment:
The indentation looks a bit weird. Better start on the first line.

Reply:
I am sorry but I do not understand this comment.

I rearranged the indentation manually to make this look like a hand-totalled sum. Is that not effective?